### DISCUSSION TITLE
Are the other blocks from the past permanently gone?

### DISCUSSION THREAD

**Discussion by plvkyan** [shadcn-ui/ui#5468] [Q&A]
Hi. I've been using shadcn since 3 months ago and I noticed that there was a recent update adding sidebars. There were lots of new sidebars in the blocks page and I can't find the old blocks. I referred to them because of their card layout since I still haven't grasped them completely (I'm still a student). Are they gone permanently? Is there anyway to access them again? Thank you in advance!

**Comment by dabreyna** [comment]
Come here for the same question, hopefully they bring them back :(

**Reply by plvkyan** [reply]
Well, there is this [tweet](https://x.com/shadcn/status/1848032433771511961). I understand it might take time but I do hope they are brought back quickly. I love the new updates though.
